migrations/versions/0451_create_db_users.py

```diff
@@ -0,0 +1,45 @@
+"""
+
+Revision ID: 0451_create_db_users
+Revises: 0450_enable_pinpoint_provider
+Create Date: 2024-05-23 12:00:00
+
+"""
+from alembic import op
+
+revision = "0451_create_db_users"
+down_revision = "0450_enable_pinpoint_provider"
+
+super_role = "rds_superuser"
+roles = ["app_db_user", "quicksight_db_user"]
+database_name = op.get_bind().engine.url.database  # database name that the migration is being run on
+
+
+def upgrade():
+    # Skip this migration in the test database as there are multiple test databases that are created.
+    # This leads to a race condition attempting to alter the same users\ multiple times and causes
+    # sporadic unit test failures.
+    if "test_notification_api" in database_name:
+        return
+
+    for role in roles:
+        create_role_if_not_exist(role)
+        op.execute(f"GRANT {role} TO {super_role} WITH ADMIN OPTION;")
```

Comment:
My bad, the above will also not work, but this will:
```python
    create_role_if_not_exist(super_role)
    for role in roles:
        create_role_if_not_exist(role)
        op.execute(f"GRANT {role} TO {super_role} WITH ADMIN OPTION;")
```